src/Rule/UrlHandler.php

```diff
@@ -39,13 +39,7 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
             }
         }
 
-        $result->addError(
-            $rule->getMessage(),
-            [
-                'attribute' => $context->getAttribute(),
-                'value' => $value,
```

Comment:
`value` can be useful for custom messages.

Reply:
Currently these are escaped mutants. See new related issue - #362. 

I suggest to handle it separately. And with "attribute" and "value" - possibly on the upper level. Maybe go even further and allow user to customize which parameters are passed.

And this is inconsistent with other rules - only few uses them while others don't.

Reply:
Discussed in group chat and decided to handle it separately.

Reply:
> Discussed in group chat and decided to handle it separately.

Ok, but then there is no need to change it within this PR.

Reply:
Discussed it in PM and decided to revert these changes.